test/language/statements/try/early-catch-duplicates.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-try-statement-static-semantics-early-errors
+es6id: 13.15.1
+description: >
+    It is a Syntax Error if BoundNames of CatchParameter contains any duplicate
+    elements.
+negative: SyntaxError
+---*/
+
+$ERROR('This code should not be executed.');
```

Comment:
[Still others](https://github.com/tc39/test262/blob/c230b7f307ecc33974b79998638496cd27039bb3/test/language/directive-prologue/10.1.1-8gs.js) use `NotEarlyError`. I agree that we could use some consistency, but my long-term goal is to avoid the concern altogether. See the discussion at gh-542 for details on how encoding the error "stage" in the frontmatter would take this burden off of test writers.